demos/browser/README.md

```diff
@@ -1,18 +1,29 @@
-### Running the demos
+## Browser Meeting
 
-To run the `meeting` application demo locally:
+This demo shows how to use the Amazon Chime SDK to build meeting applications for browsers.
 
-1. Ensure you have AWS credentials configured in your `~/.aws` folder for a role with a policy allowing `chime:CreateMeeting`, `chime:DeleteMeeting`, and `chime:CreateAttendee`.
+### Prerequisites
 
-2. Change to the `demos/browser` folder: `cd demos/browser`
+To build, test, and run demos from source you will need:
+
+* Node 10 or higher
+* npm 6.11 or higher
+
+### Running the browser demos with a local server
+
+1. Ensure you have AWS credentials configured in your `~/.aws` folder for a
+role with a policy allowing `chime:CreateMeeting`, `chime:DeleteMeeting`, and
+`chime:CreateAttendee`.
+
+2. Navigate to the `demos/browser` folder: `cd demos/browser`
 
 3. Start the demo application: `npm run start`
 
 4. Open http://localhost:8080 in your browser.
 
 ### Demo applications
 
-Demo applications are located in the `app` folder. Current demos are:
+Browser demo applications are located in the `app` folder. Current demos are:
 
 * `meeting` - incorporates all functionality into a videoconferencing application with a Bootstrap user interface
 * `meetingV2` - A similar demo app to `meeting` with content share functionality 
```

Comment:
Should we mention this is the default when running npm run start?

Reply:
A good suggestion - Hope we can deprecate `meeting` soon and use it only for legacy integration tests.